molgenis-integration-tests/src/test/java/org/molgenis/integrationtest/data/elasticsearch/query/ElasticSearchLikeQueryIT.java

```diff
@@ -0,0 +1,26 @@
+package org.molgenis.integrationtest.data.elasticsearch.query;
+
+import org.molgenis.integrationtest.data.abstracts.query.AbstractLikeQueryIT;
+import org.molgenis.integrationtest.data.elasticsearch.AbstractElasticsearchTestConfig;
+import org.molgenis.integrationtest.data.elasticsearch.query.ElasticSearchLikeQueryIT.LikeElasticSearchTestConfig;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.testng.annotations.Test;
+import java.text.ParseException;
+
+@ContextConfiguration(classes = LikeElasticSearchTestConfig.class)
+public class ElasticSearchLikeQueryIT extends AbstractLikeQueryIT
+{
+	@Configuration
+	public static class LikeElasticSearchTestConfig extends AbstractElasticsearchTestConfig
+	{
+	}
+
+	@Override
+	// @Test Disabled, PostgreSql supports more datatypes for the like query than elastic search does
```

Comment:
Instead of skipping the complete test, you can specify which methods to call manually (instead of calling ```super.testIt()```).